### PR TITLE
Add ckecli commands to show sabakan and reboot-queue status

### PR DESCRIPTION
**Pull request by yokaze**
Signed-off-by: Daichi Sakaue <daichi-sakaue@cybozu.co.jp>

### CODE REVIEW

File: docs/ckecli.md [hunk 1]
```diff
@@ -270,6 +272,10 @@ Copy files between hosts via scp.
 
 Enable/Disable processing reboot queue entries.
 
+### `ckecli reboot-queue is-enabled`
+
+Show reboot queue is enabled or disabled.
```
Comment (morimoto-cybozu) [inline docs/ckecli.md:277]: Please clarify what is displayed.
`enabled`/`disabled`?
`true`/`false`?

File: pkg/ckecli/cmd/reboot_queue_is_enabled.go [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+	"fmt"
+
+	"github.com/cybozu-go/well"
+	"github.com/spf13/cobra"
+)
+
+var rebootQueueIsEnabledCmd = &cobra.Command{
+	Use:   "is-enabled",
+	Short: "show reboot queue status",
+	Long:  `Show reboot queue status.`,
```
Comment (morimoto-cybozu) [inline pkg/ckecli/cmd/reboot_queue_is_enabled.go:14]: The word "status" seems ambiguous.

```suggestion
	Long:  `Show whether the processing of the reboot queue is enabled or not.  "true" if enabled.`,
```

File: pkg/ckecli/cmd/reboot_queue_is_enabled.go [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+	RunE: func(cmd *cobra.Command, args []string) error {
+		well.Go(func(ctx context.Context) error {
+			disabled, err := storage.IsRebootQueueDisabled(ctx)
+			fmt.Println(!disabled)
+			return err
```
Comment (morimoto-cybozu) [inline pkg/ckecli/cmd/reboot_queue_is_enabled.go:20]: Handle `err != nil` before printing.
We cannot say "true" or "false" when `err != nil`.

**Review by morimoto-cybozu — CHANGES_REQUESTED** [review]

**Comment by yokaze** [comment]
This PR should be merged with squash.

**Review by morimoto-cybozu — APPROVED** [review]
LGTM

**Pull request merged**